amulet/helpers.py

```diff
@@ -65,9 +68,9 @@ def juju(args, env=None, include_model=True):
             if arg.startswith(model_flag):
                 break
         else:
-            args = list(args)
-            # insert the model arg after the command, but before any other args
-            args[1:0] = [model_flag, default_environment()]
+            if env is None:
+                env = os.environ
+            env['JUJU_ENV'] = env['JUJU_MODEL'] = default_environment()
```

Comment:
Since we're relying on the model to come from an envar, can we pull out the `model_flag` bits and collapse this `for/else` block?